### DISCUSSION TITLE
CNF WG - High Level Roadmap / Ambition for 2021

### DISCUSSION THREAD

**Discussion by Smitholi67** [lfn-cnti/bestpractices#77] [General]
Is there anyone else that feels it might help us to create a high-level roadmap for 2021 with some key goals/objectives?  The point is less about setting a fixed schedule, and more about helping to increase our focus.  As an example (and discussion/improvement) I was thinking of something along the lines of:

**2021**

**Q1** 
-  Main ways of working and governance established 
-  Co-chairs and Tech Leads finalized
-  _x_ use cases/user stories developed

**Q2** 
- First _x_ best practices established, agreed, and shared with CNF Test Suite  
- Kubecon Europe - Share first best practices
- Next _x_ use cases/user stories developed

**Q3**
- Next x best practices established, agreed, and shared with CNF Test Suite
- Kubecon North America - Share update on best practices

....

**Comment by iawells** [comment]
I think we should expect to publish baseline 1 by (say) Q3.  Perhaps compliance test recommendations per use case by Q4?

**Reply by Smitholi67** [reply]
That is certainly valid @iawells .  My thinking was to establish a high-level ambition for 2021 so we would create more structure as well as a better understanding of what needs to be progressed and by when.  The point is not to create a rigid project plan, but to outline some key goals/milestones that we can agree upon.  Otherwise we attend one meeting after another without any agreed view on whether or not we are progressing as per the group's expectations.

**Comment by Smitholi67** [comment]
@iawells - In reference to setting some type of high-level plan/ambition I opened this back in March.  You commented and then it unfortunately never gained any steam.   I think as a group, we could have something like this ....i.e. lightweight, but providing us with some group agreed ambitions/outcomes.  Please note, the original example has not been updated so it is certainly no longer accurate.
